src/terraform.js

```diff
@@ -36,8 +36,25 @@ const init = async (opts = {}) => {
 };
 
 const apply = async (opts = {}) => {
-  const { dir = './' } = opts;
-  return await exec(`terraform -chdir='${dir}' apply -auto-approve`);
+  const { dir = './', json = false } = opts;
+  if (!json) return await exec(`terraform -chdir='${dir}' apply -auto-approve`);
```

Comment:
I do not fully understand why we keep the non json exec if we are not giving any chance to use it aside unit testing

Reply:
> we could change that one as well.

Yes, well indeed we do not have to change that... we just only need to make apply behave one way